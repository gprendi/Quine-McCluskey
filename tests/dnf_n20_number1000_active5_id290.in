d 20
00101110001000000000
00101100000000000000
00101000000000000000
00101000001000000000
00100100000000000000
00101110000000000000
00100000000000000000
00000100001000000000
00001110001000000000
00001100000000000000
00101010001000000000
00100110000000000000
00101100001000000000
00000110001000000000
00100110001000000000
00000010000000000000
00001010001000000000
00100000001000000000
00100010000000000000
00000110000000000000
00000100000000000000
00001000000000000000
00000010001000000000
00101010000000000000
00100010001000000000
00100100001000000000
00001110000000000000
00001010000000000000
00000000000000000000
00000000001000000000
00001000001000000000
00001100001000000000
