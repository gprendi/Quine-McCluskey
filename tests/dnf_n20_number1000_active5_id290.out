p dnf 20 0
-1 -2 -4 -8 -9 -10 -12 -13 -14 -15 -16 -17 -18 -19 -20 0
